### PR TITLE
cpu: use hwtimer_compat for nrf51,sam3,lcp1768 and atmega2560

### PR DESCRIPTION
**Pull request by haukepetersen**

### CODE REVIEW

**Comment by kaspar030** [comment]
Nice. ACK after travis.

File: cpu/atmega2560/Makefile.include [hunk 1]
```diff
@@ -1,9 +1,12 @@
-
 # this CPU implementation is using the new core/CPU interface
 export CFLAGS += -DCOREIF_NG=1
 
 # tell the build system that the CPU depends on the atmega common files
-export USEMODULE += atmega_common
+USEMODULE += atmega_common
+# use hwtimer compatibility module
```
Comment (kaspar030) [inline cpu/atmega2560/Makefile.include]: actually I like spaces here...

**Comment by PeterKietzmann** [comment]
Just a side question: did you even test the hwtimer on atmega2560? I'm asking because of #3186

**Comment by haukepetersen** [comment]
nope. But as I didn't actually change the code (just moved it if you want to call it like this), I did not change any behavior of that board (not for the worse but also not for the better...).

**Comment by haukepetersen** [comment]
Travis is happy -> go.

**Pull request merged**